cmd/fsdiff/main.go

```diff
@@ -84,4 +88,9 @@ func main() {
 	if err != nil {
 		log.Fatalf("write summary to disk: %v", err)
 	}
+
+	if *args.verbose {
+		debug.PrintSummarySummary(s)
+		debug.PrintDiffSummary(d)
+	}
```

Comment:
I find this is helpful when running it as an operator to understand if it is doing what you think its doing!